portnums.proto

```diff
@@ -134,6 +134,12 @@ enum PortNum {
    * in your code without needing to rebuild protobuf files (via [regen-protos.sh](https://github.com/meshtastic/firmware/blob/master/bin/regen-protos.sh))
    */
   PRIVATE_APP = 256;
+  
+  /*
+   * Black Lager encryption  port
```

Comment:
Consider adding more description of what the packets sent over the Black Lager port too. Mention it is a text messaging application with digital signatures.